### PR TITLE
[12.0][IMP] sale_automatic_workflow_job - related_job

### PR DESCRIPTION
**Pull request by Saran440**
Fixed related from job to origin document
Selection_002

### CODE REVIEW

**Review by ps-tubtim — APPROVED** [review]
Code LGTM

**Review by marcelsavegnago — APPROVED** [review]

File: sale_automatic_workflow_job/models/queue_job.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import _, models
+
+
+class QueueJob(models.Model):
+    """ Job status and result """
+    _inherit = "queue.job"
+
+    def _related_action_sale_automaitc_workflow(self):
```
Comment (guewen) [inline sale_automatic_workflow_job/models/queue_job.py]:
```suggestion
    def _related_action_sale_automatic_workflow(self):
```

Typo in the name (to correct in the references too)

**Review by guewen — CHANGES_REQUESTED** [review]
Thanks, nice improvement.

**Comment by Saran440** [comment]
@guewen Fixed. Thank you for your reviewed.

**Review by guewen — APPROVED** [review]
Thanks!

**Pull request merged**